.changeset/tricky-avocados-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@urql/exchange-graphcache': major
+---
+
+The deprecated `cache.invalidateQuery()` method has been removed. Please migrate over to `cache.invalidate()` instead, which operates on individual fields instead of queries.
```

Comment:
Note: this is a separate log because it’s an extremely important point for migration, so we should separate it out from the other changeset